config.json

```diff
@@ -1172,6 +1172,15 @@
         "Recursion",
         "Searching"
       ]
+    },
+    {
```

Comment:
The ordering of the core exercises is important. Could you move this exercise's entry up, I think between the `sum-of-multiples` and `raindrops` exercises.